buf.work.yaml

```diff
@@ -1,4 +1,3 @@
 version: v1
 directories:
   - proto
-  - pkg/motor/proto
```

Comment:
?

Reply:
that directory got moved out and running `buf build` would error